### PR TITLE
Add Johnson Fitness & Wellness

### PR DESCRIPTION
**Pull request by molliemchugh-tomtom**
Hello,

I'm adding Johnson Fitness & Wellness.

https://www.johnsonfitness.com/

### CODE REVIEW

File: data/brands/shop/sports.json [hunk 1]
```diff
@@ -6,6 +6,18 @@
...
+        {
+      "displayName": "Johnson Fitness & Wellness",
+      "locationSet": {
+        "include": ["au","br","ca","cn","cz","gr","hk","it","jp","kr","my","mx","ro","tw","th","uk","us","vn"]
+      },
+      "tags": {
+        "brand": "Johnson Fitness & Wellness",
+        "brand:wikidata": "Q120785852",
+        "name": "Johnson Fitness & Wellness",
+        "shop": "sports"
```
Comment (AntiCompositeNumber) [inline data/brands/shop/sports.json:18]:
```suggestion
        "shop": "sports",
        "sport": "fitness"
```
Based on their website, I think it makes sense to add [`sport=fitness`](https://wiki.openstreetmap.org/wiki/Tag%3Asport%3Dfitness)

**Comment by molliemchugh-tomtom** [comment]
Hello @AntiCompositeNumber,

This company sells sports equipment.  It is not a fitness center or gym.

I think it should be left as is.

**Comment by AntiCompositeNumber** [comment]
`sport=` is commonly used to specify the type of equipment a `shop=sports` sells. I'm not particularly opposed to leaving it out, but I do think it would be useful to include.

**Comment by molliemchugh-tomtom** [comment]
My understanding is that sports=fitness is for locations where people use sports equipment to improve their health/fitness.

https://wiki.openstreetmap.org/wiki/Tag%3Asport%3Dfitness#:~:text=The%20tag%20sport%3Dfitness%20is%20commonly%20added%20to%20features%20such%20as%20leisure%3Dfitness_centre%20or%20leisure%3Dfitness_station%20where%20people%20develop%20physical%20fitness%20by%20group%20classes%2C%20cardio%20machines%2C%20weight%20lifting%2C%20and%20related%20activities

This business is a place where sports equipment/workout machines **are sold** .  It is not a club, center, or location for using sports equipment.

However, if you insist on changing this to shop=fitness, I will do so.

File: data/brands/shop/sports.json [hunk 2]
```diff
@@ -6,6 +6,18 @@
     }
   },
   "items": [
+        {
+      "displayName": "Johnson Fitness & Wellness",
+      "locationSet": {
+        "include": ["au","br","ca","cn","cz","gr","hk","it","jp","kr","my","mx","ro","tw","th","uk","us","vn"]
+      },
```
Comment (AntiCompositeNumber) [inline data/brands/shop/sports.json:13]:
```suggestion
      },
      "matchNames": ["johnson fitness & wellness store"],
      "matchTags": ["leisure/fitness_centre"],
```

**Review by AntiCompositeNumber — CHANGES_REQUESTED** [review]
Looking at https://overpass-turbo.eu/s/1Dc7, looks like there's 2 stores with `sport=fitness`, but I'm fine leaving that out.

There are a bunch of stores tagged `leisure=fitness_centre` `name=Johnson Fitness & Wellness Store` so matchNames and matchTags should be added for those.

**Review by AntiCompositeNumber — APPROVED** [review]

**Comment by Cj-Malone** [comment]
Thanks everyone

**Pull request merged**